Source/WebKit/PlatformWPE.cmake

```diff
@@ -356,8 +356,14 @@ add_custom_command(
     VERBATIM
 )
 
-set(WebKitResources
-)
+set(WebKitResources "")
+list(APPEND WebKitResources "<file alias=\"css/wpe-theme.css\">wpe-theme.css</file>\n")
+list(APPEND WebKitResources "<file alias=\"images/missingImage\">missingImage.png</file>\n")
+list(APPEND WebKitResources "<file alias=\"images/missingImage@2x\">missingImage@2x.png</file>\n")
+list(APPEND WebKitResources "<file alias=\"images/missingImage@3x\">missingImage@3x.png</file>\n")
+list(APPEND WebKitResources "<file alias=\"images/panIcon\">panIcon.png</file>\n")
+list(APPEND WebKitResources "<file alias=\"images/textAreaResizeCorner\">textAreaResizeCorner.png</file>\n")
+list(APPEND WebKitResources "<file alias=\"images/textAreaResizeCorner@2x\">textAreaResizeCorner@2x.png</file>\n")
```

Comment:
Why don't you just set in a single command instead?

Reply:
I used `PlatformGTK.cmake` as reference. I don't know the reason why in `PlatformGTK.cmake` was done this way, but I suspect it had to do with a lint error returned by `Tools/Scripts/check-webkit-style`. I managed to solve this lint error using a single list, so I changed `PlatformGTK.cmake` too.